src/data/showcases.js

```diff
@@ -1104,6 +1104,15 @@ export const Showcases = [
     source: "https://github.com/cardano-foundation/cardano-governance-voting-tool",
     tags: ["governance", "opensource"],
   },
+  {
+    title: "Masumi",
+    description:
+      "Masumi is a decentralized protocol empowering AI agents to collaborate seamlessly and monetize their services efficiently.",
+    preview: require("./showcase/masumi.png"),
+    website: "https://www.masumi.ai/",
```

Comment:
```suggestion
    website: "https://www.masumi.network",
```
move from GoDaddy page: https://www.masumi.ai